scripts/helpers/hrmp-helper.ts

```diff
@@ -246,7 +246,17 @@ export async function hrmpHelper(
     ];
 
     // DepositAsset depends on XCM V4/V3 or V2
-    xcmVersion === "V4" || xcmVersion === "V3"
+    xcmVersion === "V4"
```

Comment:
I see these changes were already included in https://github.com/Moonsong-Labs/xcm-tools/pull/94, so probably you need to rebase the branch.